FBSnapshotTestCase/FBSnapshotTestCase.m

```diff
@@ -111,17 +112,52 @@ - (NSString *)snapshotVerifyViewOrLayer:(id)viewOrLayer
       }
     }
   }
-  
-  if (!testSuccess) {
-    return [NSString stringWithFormat:@"Snapshot comparison failed: %@", errors.firstObject];
-  }
+    
+  [self addAttachementsWithErrors:errors identifier:identifier];
+    
   if (self.recordMode) {
-    return @"Test ran in record mode. Reference image is now saved. Disable record mode to perform an actual snapshot comparison!";
+    if (errors.count > 0) {
+      return [NSString stringWithFormat:@"Snapshot comparison failed: %@", errors.firstObject];
+    } else {
+      return @"Test ran in record mode. Reference image is now saved. Disable record mode to perform an actual snapshot comparison!";
+    }
+  } else if (!testSuccess) {
+    return [NSString stringWithFormat:@"Snapshot comparison failed: %@", errors.firstObject];
   }
 
   return nil;
 }
 
+- (void) addAttachementsWithErrors:(NSArray<NSError*>*)errors identifier:(NSString*)identifier {
+#ifdef __IPHONE_11_0
+    if (self.recordMode) {
+        UIImage* image = [_snapshotController referenceImageForSelector:self.invocation.selector identifier:identifier error:nil];
```

Comment:
Most of the repo uses a different style for asterisk–alignment, I think you should match it.